README.md

```diff
@@ -42,6 +42,11 @@ Make sure to checkout the repository (actions/checkout@v2) to provide the ZAP ru
 **Optional** ZAP action uses the default action token provided by GitHub to create and update the issue for the baseline scan.
 You do not have to create a dedicated token. Make sure to use the GitHub's default action token when running the action(`secrets.GITHUB_TOKEN`).
 
+### `fail_action`
+
+**Optional** By default ZAP Docker container will fail with an [exit code](https://github.com/zaproxy/zaproxy/blob/develop/docker/zap-baseline.py#L32-L35), 
```

Comment:
We should link to actual commit (or tag), otherwise this might break (e.g. branch renamed, docs/code moved around).

Reply:
Okay, Got the commit: https://github.com/zaproxy/zaproxy/blob/efb404d38280dc9ecf8f88c9b0c658385861bdcf/docker/zap-baseline.py#L31